### DISCUSSION TITLE
Significance of orphanedJobs which is handled in the Job Runner Code

### DISCUSSION THREAD

**Discussion by bhushanibm** [jobrunr/jobrunr#531] [Q&A]
I see there is way to set the orphanedJobsRequestSize before we start the background job server. What does it signify ? 
Does setting this size will look for job which were not fired before ( may be pod was down) and will try to trigger them ?

**Comment by bhushanibm** [comment]
@rdehuyss  :- Can you please clarify the above since we are looking for feature around jobs which did not trigger since jobrunner pod was down at that moment ? Will setting the above will help us ?

**Reply by bhushanibm** [reply]
@rdehuyss :- Can you please clarify the above since we are looking for feature around jobs which did not trigger since jobrunner pod was down at that moment ? Will setting the above will help us ?

**Reply by rdehuyss** [reply]
@bhushanibm : feel free to have a look in the code. This is a community supported discussion board. Tagging me won't help you get answers faster. And it's not because it is open-source that I work for free...

**Reply by rdehuyss** [reply]
If you want support, feel free to reach out for a JobRunr Pro subscription.
